tables/hbo.tbl

```diff
@@ -0,0 +1,263 @@
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+#The first Hebrew Braille table hosted on Lib Louis was developed by the Library for the Blind and the Ministry of Education in Israel. That  table is based on the IHBC which was developed in the mid-1930’s. Please see World Braille usage 3rd edition, p. 74. 
+#However, Classical or Biblical Hebrew, which is used in the Hebrew Bible and other liturgical literature, contains cantillation marks that are not supported by the Hebrew table mentioned above. This table seeks to provide access to these accents. This table is   also based upon the tables mentioned above.  This table only departs from it where the accents are concerned. With this table, the user will have access to the accents and Masorah in BHS and BHS Quinta. For more information about the IHBC and How it was developed:
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+letter שְּׁ 146-3
+letter שְּׂ 5-156-3
+letter פְּ 5-1234-3
+letter תְּ 2345-3
+letter מְּ 5-134-3
+letter כְּ 13-3
+letter נְּ 5-1345-3
+letter יְּ 5-245-3
+letter קְּ 5-12345-3
+letter ּוְ 136-3
+letter דְּ 5-145-3
+letter שְׁ 146-3
+letter בְּ 12-3
+letter שּׁ 146-3
+letter שּׂ 5-156
+letter שְׂ 156-3
+letter לְּ 5-123-3
+letter ש 146
+letter הּ 125
+letter קּ 5-12345
+letter ר 1235
+letter וֹ 246
+letter וּ 136
+letter ּו 136
+letter ֵי 3456
+letter יּ 5-245
+letter שׁ 146
+letter פּ 5-1234
+letter ִי 35
+letter כּ 13
+letter נּ 5-1345
+letter כְּ 13-3
+letter לּ 5-123
+letter דּ 5-145
+letter בּ 12
+letter בְּ 12-3
+letter תְּ 5-1456-3
+letter מּ 5-134
+letter שׂ 156
+letter תּ 2345
+letter ק 12345
+sign ✙ 23456-6
+letter צ 2346
+sign ❑ 12345
+sign ✔ 12356
+sign ❦ 6-156
+sign ✗ 24
+sign ❥ 16-6
+sign ✘ 35
+letter ץ 2346
+sign ❚ 14
+sign ❁ 123-1346-1346
+sign ❘ 1256
+sign ❍ 135
+sign ❏ 135-6
+sign ❙ 2345
+sign ❐ 1235
+sign ✕ 1235
+sign ✖ 2456
+sign ❧ 156-6
+letter פ 1234
+letter ם 134
+letter ֤ 46-125
+letter ֫ 46-135
+letter ֪ 46-13456
+letter ֩ 46-12345
+letter ֨ 46-1234
+letter ֧ 46-145
+letter ֦ 46-13
+letter ֥ 46-134
+letter ֣ 46-136
+letter ֭ 46-24
+letter ֢ 46-23
+letter ֡ 46-236
+letter ֠ 46-2356
+letter ֟ 46-1345
+letter ֞ 46-12
+letter ֝ 46-16
+letter ֜ 46-1
+letter ֛ 46-1236
+letter ֬ 46-24
+letter ֮ 46-1356
+letter ל 123
+letter ֹ 246
+letter ֲ 25
+letter ֳ 345
+letter ִ 24
+letter ֵ 34
+letter ֶ 15
+letter ַ 14
+letter ָ 126
+letter ֺ 5-246
+letter ֯ 456-134
+letter ֻ 346
+letter ּ 5
+letter ֽ 4
+punctuation ־ 36
+letter ֿ 2
```

Comment:
Is this a rafe? (that's the little line that goes above the vet and fey in some Yiddish texts). 

I wouldn't ordinarily have expected a biblical Hebrew braille table to include Yiddish-specific letters, but since this table already has the Yiddish-specific digraphs װ and ױ and ײ, it seems like all it needs is the ability to handle בֿ, פֿ, and ײַ and then it would be able to handle Yiddish texts, too, which I think would be wonderful.

Reply:
If you are going to add בֿ (vet with rafe), I would use 1236, the same as a vet without the rafe. in Yiddish, bet (without dagesh) is very common and used to make a "b" sound, and vet with dagesh is used to make a "v" sound, but it only appears in words borrowed from ancient Hebrew. 

Fey with rafe, פֿ I would use 124, the same as fey without the rafe. If you want to write the rafe separately in this table to be extra clear about what dots are what, then I don't really have a suggestion - the rafe seems to indicate that the consonant is a fricative, as opposed to the dagesh which indicates that the consonant is a stop. But I don't know if there are any existing braille dots that have that meaning. I think the important thing is just to make sure that a word with a vet-with-rafe renders the same way as a vet, and that fey-with rafe renders the same way as a fey. I have never seen rafe used with any other letters, but maybe they appear occasionally in the biblical Hebrew texts you're working with?